frontend/src/components/pages/Account.tsx

```diff
@@ -0,0 +1,130 @@
+import {
+  Box,
+  Button,
+  Container,
+  FormControl,
+  HStack,
+  IconButton,
+  Input,
+  Text,
+} from "@chakra-ui/react";
+import React, { useContext, useState } from "react";
+import { useHistory } from "react-router-dom";
+
+import DonorAPIClient from "../../APIClients/DonorAPIClient";
+import { DASHBOARD_PAGE } from "../../constants/Routes";
+import AuthContext from "../../contexts/AuthContext";
+import { Role } from "../../types/AuthTypes";
+
+const Account = (): JSX.Element => {
+  const history = useHistory();
+  const { authenticatedUser, setAuthenticatedUser } = useContext(AuthContext);
+  const [businessName, setBusinessName] = useState("");
+
+  const navigateToDashboard = () => {
+    history.push(DASHBOARD_PAGE);
+  };
+
+  React.useEffect(() => {
+    if (!authenticatedUser) {
+      return;
+    }
+    const getDonor = async () => {
+      const donor = await DonorAPIClient.getDonorByUserId(authenticatedUser.id);
+      setBusinessName(donor.businessName);
+    };
+    getDonor();
+  }, [authenticatedUser]);
+
+  if (!authenticatedUser) {
+    return <></>;
+  }
+
+  return (
+    <>
+      <Container pl="42px" pr="42px" pt="0.5rem">
+        <Text mt="67px" textStyle="mobileHeader1">
+          My Account
+        </Text>
+        <FormControl mt="2rem" isReadOnly>
+          <Text mb="1em" textStyle="mobileBodyBold" color="hubbard.100">
+            Organization
+          </Text>
+          <Box>
+            <Text>Name of business</Text>
+            <Input
+              mt="2"
+              value={businessName}
+              name="businessName"
+              placeholder="Enter name of business"
+              variant="unstyled"
+              size="sm"
+            />
+          </Box>
+          <Text
+            mt="2rem"
+            mb="1em"
+            textStyle="mobileBodyBold"
+            color="hubbard.100"
+          >
+            Point of Contact
+          </Text>
+          <HStack>
+            <Box>
+              <Text>First name</Text>
+              <Input
+                mt="2"
+                value={authenticatedUser!.firstName}
+                name="firstName"
+                placeholder="Enter first name"
+                variant="unstyled"
+                size="sm"
+              />
+            </Box>
+            <Box mt="1rem">
```

Comment:
```suggestion
            <Box mt="1.5rem">
```
can we make this a bit larger?

Reply:
fixed!